### PR TITLE
English Documentation

### LINKED ISSUE #1: Create English Documentation
Create the english documentation in Readme.english.md file.

### PR DESCRIPTION
**Pull request by willy23martin**
Create English documentation.

### CODE REVIEW

**Comment by willy23martin** [comment]
Fix Spanish README.md Spanish version and complete its English version

**Comment by willy23martin** [comment]
Could you please give me an approval to my English documentation pull request?

**Comment by willy23martin** [comment]
Could you please give me an approval to my English documentation pull request? https://github.com/icesi-devops/edu2020b-python/pull/11

File: English/README.md [hunk 1]
```diff
@@ -84,36 +143,61 @@
...
+    * Request For Changes (RFC) validation before deployment.
+  * Continuous Integration tools
+    * [Travis CI](https://travis-ci.org/).
+    * GitHub Actions.
+    * [DroneIO](https://cloud.drone.io/).
+    * [CircleCI](https://circleci.com/).
+    * [AWS CodePipeline](https://aws.amazon.com/es/codepipeline/).
+  * Implement Python code scripts for pipeline stages.
+  * **ChatOps implementation**
+    * [Microsoft Teams]().
```
Comment (d4n13lbc) [inline English/README.md:183]: Could you please add discord here, I will add it to the Spanish version later

Comment (willy23martin) [reply English/README.md:183]: Done.

**Review by d4n13lbc — APPROVED** [review]
LGTM

**Pull request merged**